tests/delorean_tests.py

```diff
@@ -773,5 +773,13 @@ def test_humanize_future(self, mock_now):
         do = delorean.Delorean(datetime(2015, 1, 1), timezone='US/Pacific')
         self.assertEqual(do.humanize(), 'a day from now')
 
+    def test_replace(self):
+        do = delorean.Delorean(datetime(2015, 1, 1), timezone='US/Pacific')
+        self.assertEqual(do.replace(hour=8).datetime.hour, 8)
+        self.assertEqual(do.replace(minute=23).datetime.minute, 23)
+        self.assertEqual(do.replace(second=45).datetime.second, 45)
+
+
```

Comment:
I would also like to see a few asserts on the entire datetime object as well and also assuring the timezone is unchanged.